operator/v1/types_ingress.go

```diff
@@ -545,6 +545,22 @@ type AWSLoadBalancerParameters struct {
 	//
 	// +optional
 	NetworkLoadBalancerParameters *AWSNetworkLoadBalancerParameters `json:"networkLoadBalancer,omitempty"`
+
+	// subnets specifies the subnets for the load balancer to route
+	// traffic to. The subnets may be specified by either their
+	// ID or name. The total number of subnets is limited to 10.
+	//
+	// In order for the load balancer to be provisioned with subnets:
+	// * Each subnet must exist.
+	// * Each subnet must be from a different availability zone.
+	// * The load balancer service must be recreated to pick up new values.
+	//
+	// If omitted, the subnets will be auto-discovered per availability zone.
+	// When subnets are auto-discovered, they do not propagate to these lists.
```

Comment:
Nit, but makes things clearer:
```suggestion
	// Auto-discovered availability zone subnets are not propagated to subnets.
```

Reply:
It seems strange to have this statement at all, but I suppose it's a consequence of using the same type in `spec` and `status`—right?  The point is that _status_ does not reflect auto-discovered subnets.

Reply:
Thanks, the revised wording is clearer: "Auto-discovered subnets are not propagated in the status of this field."  I would only suggest changing "this field" to "the IngressController object" (as "this field" in the godoc for the spec field would seem to refer to the spec field), and perhaps change "propagated" to "reported" (in order to use more familiar language): "Auto-discovered subnets are not reported in the status of the IngressController object."

Reply:
Fair point. Done.